### DISCUSSION TITLE
Is it normal for the Podman Desktop tray icon to look so low quality?

### DISCUSSION THREAD

**Discussion by Luxvao** [podman-desktop/podman-desktop#834] [Q&A]
I've installed Podman Desktop on windows 10 today and I've noticed that the tray icon looks so much better than the one on windows 11. I don't have a refrence picture, but I do have the picture on windows 11:
![image](https://user-images.githubusercontent.com/95533447/201932994-fa050767-cbab-477c-9de2-ba89f92c713e.png)
Is that normal or is it a problem with windows 11?

**Comment by patstha** [comment]
for what it is worth, here is what I see in 4.3.1 installed from scoop 
![image](https://user-images.githubusercontent.com/56666486/202878553-451cc70d-d405-408f-b4c6-5816ed0877c2.png)

**Reply by Luxvao** [reply]
Thats also what I saw and it looks way better than the one I am seemingly stuck with..
